src/Execution/values.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Digia\GraphQL\Execution;
+
+use Digia\GraphQL\GraphQL;
+use Digia\GraphQL\Language\Node\ArgumentsAwareInterface;
+use Digia\GraphQL\Language\Node\VariableDefinitionNode;
+use Digia\GraphQL\Type\Definition\Directive;
+use Digia\GraphQL\Type\Definition\DirectiveInterface;
+use Digia\GraphQL\Type\Definition\Field;
+use Digia\GraphQL\Type\SchemaInterface;
+
+/**
+ * @param SchemaInterface          $schema
+ * @param VariableDefinitionNode[] $nodes
+ * @param array                    $inputs
+ * @return array
+ */
+//function getVariableValues(SchemaInterface $schema, array $nodes, array $inputs): array
```

Comment:
Why you comment this out?

Reply:
There is no such method in `ValuesHelper`.